scripts/run_bench_and_upload.sh

```diff
@@ -21,36 +14,12 @@ BENCHMARK_DATA="`pwd`/.data"
 mkdir -p ${BENCHMARK_DATA}
 BENCHMARK_FILENAME=${CIRCLE_SHA1}_$(date +"%Y%m%d_%H%M%S").json
 BENCHMARK_ABS_FILENAME=${BENCHMARK_DATA}/${BENCHMARK_FILENAME}
-pytest test_bench.py --setup-show --benchmark-sort=Name --benchmark-json=${BENCHMARK_ABS_FILENAME} -k "$PYTEST_FILTER"
-
+# configure benchmark invocation to be as fast as possible since this run isn't used for actual timing results,
+# but it's still useful to see (imprecise) timing output for debug/diagnosis of adding a new model
+# 'real' timing and score computation are handled by pytorch CI github actions runner with performance-tuned machine
+# use --benchmark-disable for now to make things fast; need to modify defaults in test_bench.py before possible to override runtime cmd line:
```

Comment:
i didn't end up using this, but what it does is just runs all models once and _not_ time them.

Since we wanted timing, and we also want a json file output to test compute_score.py, I switched to min_rounds=1, max_time=<something small> and warmup-disable.  It makes the suite run quite a bit faster but still not as fast as the python test.py case, so there is some overhead.